### PR TITLE
Adding to-do app using Flask

### PR DESCRIPTION
**Pull request by muhammadgalhoum**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md)?

**yes**

### Description

**To-do app using Flask
You should install all requirements in the requirements.txt file before running the project 
You can take a view of this project's features in the README.md file of this project**

Fixes #issue_no #525 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by muhammadgalhoum** [comment]
**I have tried to solve this problem #535, but I can't, so I have made this new pull request
Please review it @photon149**

File: Web Development/Intermediate/To-do app/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+![SignP](Images/Register.PNG)
+
+### 3. Home page
+
+![Home page](Images/Home.PNG)
+
+### 4. Create New Task (Add Todo)
+
+![Create New Task](Images/Add%20New%20Task.PNG)
```
Comment (Kushal997-das) [inline Web Development/Intermediate/To-do app/README.md:19]: Reduce the width .

File: Web Development/Intermediate/To-do app/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (Kushal997-das) [inline Web Development/Intermediate/To-do app/README.md:20]: @muhammadgalhoum

File: Web Development/Intermediate/To-do app/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+### 7. Updating Completed Task name or status
+
+![Edit Completed Task](Images/Edit%20Completed%20Task.PNG)
+
+### 8. Updating UnCompleted Task name or status
+
+![Edit UnCompleted Task](Images/Edit%20Uncompleteed%20Task.PNG)
+
+### 9. Logout
+
```
Comment (Kushal997-das) [inline Web Development/Intermediate/To-do app/README.md:38]: Reduce the width size .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@muhammadgalhoum Do the requires changes .

**Comment by muhammadgalhoum** [comment]
I did these spaces between the lines to avoid any errors or warnings in the README.md file
Do you want me to delete these spaces or what do you need? @Kushal997-das

**Comment by Kushal997-das** [comment]
> I did these spaces between the lines to avoid any errors or warnings in the README.md file Do you want me to delete these spaces or what do you need? @Kushal997-das

No man just reduce the width size of the all images which are in your `readme.md` file .

**Comment by muhammadgalhoum** [comment]
I took a complete page screenshot for each feature on my website to appear this feature clearly
Do you need me to crop each of the screenshots pages of my website to reduce its width?
@Kushal997-das

**Comment by Kushal997-das** [comment]
> I took a complete page screenshot for each feature on my website to appear this feature clearly Do you need me to crop each of the screenshots pages of my website to reduce its width? @Kushal997-das

Don't you understand what is the meaning of width ? No need to crop you can reduce the width using HTML tags in the readme file . After all changes do comment here .

**Comment by muhammadgalhoum** [comment]
I have used this way `Login` not this way` `

See my new code changes and give me your opinion about these changes
@Kushal997-das

File: Web Development/Intermediate/To-do app/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+### 4. Create New Task (Add Todo)
+
+![Create New Task](screenshots/Add%20New%20Task.PNG)
+
+### 5. Completed Tasks page
+
+![Completed Tasks](screenshots/Completed%20Tasks.PNG)
+
+### 6. Uncompleted Tasks page
```
Comment (Kushal997-das) [inline Web Development/Intermediate/To-do app/README.md:25]: Use this command to reduce image size .

`

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@muhammadgalhoum

File: Web Development/Intermediate/To-do app/README.md (hunk 4 above)
Comment (muhammadgalhoum) [reply Web Development/Intermediate/To-do app/README.md:25]: As you like

**Comment by muhammadgalhoum** [comment]
Can you please review it again @Kushal997-das

**Comment by muhammadgalhoum** [comment]
@Kushal997-das 
When I have tried to use this command ``most of the images became like the following image so I resize it using a special website and make the width 600px as you like

Capture

**Comment by Kushal997-das** [comment]
> @Kushal997-das When I have tried to use this command ``most of the images became like the following image so I resize it using a special website and make the width 600px as you like
> 
> Capture

Man you should try yourself i gave you just idea how you can resize using HTML tags . Don't crop the image as you are going to reduce the size using HTML tag , so use only original image then use these tags . YOu can also try to change height and width of the image .

**Comment by muhammadgalhoum** [comment]
Are you need all the screenshots to have the same width and height? @Kushal997-das

**Comment by muhammadgalhoum** [comment]
Please review it again @Kushal997-das

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@muhammadgalhoum What should i review any of these links are not working in your `readme.md` file .

**Comment by muhammadgalhoum** [comment]
You asked me to make all the pictures the same size and height and I did that
There is no error in the README.MD file
Therefore, I asked you to review the required changes
Where are the links that don't work?!
@Kushal997-das

**Comment by Kushal997-das** [comment]
> You asked me to make all the pictures the same size and height and I did that There is no error in the README.MD file Therefore, I asked you to review the required changes Where are the links that don't work?! @Kushal997-das

check

**Comment by muhammadgalhoum** [comment]
### Please review again @Kushal997-das

**Review by Kushal997-das — APPROVED** [review]
@photon149 Have a look .

**Review by photon149 — COMMENTED** [review]
@muhammadgalhoum Nicely Implemented , though just one small addition I would suggest is to add in your README file the steps to run your app .

**Comment by muhammadgalhoum** [comment]
**Please review again @photon149 
Thanks in advanced**

**Review by photon149 — APPROVED** [review]
@muhammadgalhoum Great 👍🏻 
Approved from my end.
No Duplicity ✅ 
@Kushal997-das @prakhar3062 have a look

**Review by prakhar3062 — APPROVED** [review]

**Pull request merged**